docs/exchanges.md

```diff
@@ -16,9 +16,9 @@ Accounts having BNB accounts use this to pay for fees - if your first trade happ
 
 Binance has been split into 3, and users must use the correct ccxt exchange ID for their exchange, otherwise API keys are not recognized.
 
-* [binance.com](https://www.binance.com/) - International users - ccxt id: `binance`
-* [binance.us](https://www.binance.us/) US based users- ccxt id: `binanceus`
-* [binance.je](https://www.binance.je/) Trading FIAT currencies - ccxt id: `binanceje`
+* [binance.com](https://www.binance.com/) - International users, ccxt id: `binance`.
+* [binance.us](https://www.binance.us/) - US based users, ccxt id: `binanceus`.
```

Comment:
I would add a "use" to these 2 lines too, so it's aligned to the last which uses "use ccxt id:"
`US based users, use ccxt id:`